### PR TITLE
showSpinners directive to toggle timepicker spinners

### PR DESCRIPTION
**Pull request by armincifuentes**
Useful for proper form alignment

### CODE REVIEW

File: template/timepicker/timepicker.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <table>
 	<tbody>
-		<tr class="text-center">
+		<tr class="text-center" ng-show="showSpinners">
```
Comment (wesleycho) [inline template/timepicker/timepicker.html]: Probably want to make it `ng-show="::showSpinners"`

File: template/timepicker/timepicker.html [hunk 2]
```diff
@@ -16,7 +16,7 @@
 			</td>
 			<td ng-show="showMeridian"><button type="button" class="btn btn-default text-center" ng-click="toggleMeridian()">{{meridian}}</button></td>
 		</tr>
-		<tr class="text-center">
+		<tr class="text-center" ng-show="showSpinners">
```
Comment (wesleycho) [inline template/timepicker/timepicker.html]: Ditto

**Comment by wesleycho** [comment]
Can you explain this further, or post a [Plunker](http://plnkr.co/) with this in action?

**Comment by armincifuentes** [comment]
Oh, sorry. I guess I shoud elaborate a bit.

I think its useful to hide the spinners in order to align vertically the timepicker with other form controls in multi column forms.

I've found it quicker to compose a picture rather than figuring out Plunker. If necessary, I'd be glad to setup a working example, just maybe later.

explanation

**Comment by wesleycho** [comment]
Ahh, I was getting confused by the HTML formatting (not sure why that file was done in tabs instead of spaces...).

This LGTM, I will merge it in shortly.

**Comment by armincifuentes** [comment]
@wesleycho 
I think it should incorporate one-time binding for this directive as well, as you suggested. Will close & open new PR shortly.

**Comment by wesleycho** [comment]
Actually, I'm already doing the cleanup and writing the documentation as we speak - it will be in shortly :) .

**Comment by armincifuentes** [comment]
Great, thanks!

**Pull request closed without merging**